Tensile/TensileCreateLibrary.py

```diff
@@ -261,7 +276,12 @@ def buildSourceCodeObjectFile(CxxCompiler, outputPath, kernelFile):
               if globalParameters["PrintCodeCommands"]:
                 print(' '.join(bundlerArgs))
               # change to use  check_output to force windows cmd block util command finish
-              subprocess.check_output(bundlerArgs, stderr=subprocess.STDOUT)
+              try:
+                out = subprocess.check_output(bundlerArgs, stderr=subprocess.STDOUT)
```

Comment:
This part is already in try section.
Do we need to use nested try here?

Reply:
Just posted an update to clean that up. Thanks!

Reply:
I found 1 more subprocess.check_output call at line 238 (before change) in TensileCreateLibrary.py.
Do we need to add print for the output of this call?

Reply:
That one should also be covered by my most recent change

Reply:
I understand that the exception is captured.
I am just talking about print2 you added for all other check_output calls.
Like this.
  print2(out)